gems/aws-sdk-core/lib/aws-sdk-sts/presigner.rb

```diff
@@ -51,7 +51,7 @@ def get_caller_identity_presigned_url(options = {})
         )
 
         url = Aws::Partitions::EndpointProvider.resolve(
-          req.context.config.region, 'sts'
+          req.context.config.region, 'sts', false
```

Comment:
Thanks for the contribution.

I think this should be set to `'regional'` instead of `false` as it should be a string type: https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-partitions/lib/aws-partitions/endpoint_provider.rb#L42